boards/arm/nucleo_f334r8/doc/nucleof334r8.rst

```diff
@@ -0,0 +1,182 @@
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| IWDG      | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/nucleo_f334r8/nucleo_f334r8_defconfig``
+
+Connections and IOs
+===================
+
+Each of the GPIO pins can be configured by software as output (push-pull or open-drain), as
+input (with or without pull-up or pull-down) or as peripheral alternate function. Most of the
+GPIO pins are shared with digital or analog alternate functions. All GPIOs are high current
+capable except for analog inputs.
+
+Board connectors:
+-----------------
+.. image:: img/nucleo_f334r8_connectors.png
+     :width: 800px
+     :align: center
+     :height: 619px
+     :alt: Nucleo F334R8 connectors
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_1_TX : PA9
+- UART_1_RX : PA10
+- UART_2_TX : PA2
+- UART_2_RX : PA3
+- PWM_1_CH1 : PA8
+- USER_PB   : PC13
+- LD2       : PA5
+
+For mode details please refer to `STM32 Nucleo-64 board User Manual`_.
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+Nucleo F334R8 board includes an ST-LINK/V2-1 embedded debug tool interface.
+This interface is supported by the openocd version included in Zephyr SDK.
+
+Flashing an application to Nucleo F334R8
+----------------------------------------
+
+The sample application :ref:`blinky` is being used in this tutorial:
+
+.. code-block:: console
+
+   $<zephyr_root_path>/samples/basic/blinky
+
+To build the Zephyr kernel and application, enter:
+
+.. code-block:: console
+
+   $ cd <zephyr_root_path>
+   $ source zephyr-env.sh
+   $ cd $ZEPHYR_BASE/samples/basic/blinky
+   $ make BOARD=nucleo_f334r8
+
+Connect the Nucleo F334R8 to your host computer using the USB port.
+Then, enter the following command:
+
+.. code-block:: console
+
+   $ make BOARD=nucleo_f334r8 flash
+
+Run a serial host program to connect with your Nucleo board:
+
+.. code-block:: console
+
+   $ minicom -D /dev/ttyACM0
+
+You should see LED diode blinking every 1 second.
+
+Debugging
+=========
+
+Access gdb with the following make command:
+
+.. code-block:: console
+
+   $ make BOARD=nucleo_f334r8 debug
+
+
+References
```

Comment:
delete this heading and underline (none of the item listed below will show up in the output)